biofefi/components/forms.py

```diff
@@ -239,6 +239,20 @@ def fi_options_form():
             key=ConfigStateKeys.NumberOfTopRules,
         )
 
+    save_plots = st.checkbox(
```

Comment:
If you look at the ML page now that it's merged, do you think you could put all the options to save outputs at the bottom like on the ML page. Also could you use `st.toggle` for those options, like on the ML page pls?